.chant/specs/2026-01-30-001-vpz.md

```
@@ -0,0 +1,213 @@
+**Location:** `src/bun.js/bindings/ModuleLoader.cpp:573-618`
+
+The function is called from `BunProcess.cpp` when `process.getBuiltinModule(moduleName)` is invoked:
+
+```cpp
+JSValue resolveAndFetchBuiltinModule(
+    Zig::GlobalObject* globalObject,
+    BunString* specifier)
+{
+    void* bunVM = globalObject->bunVM();
+    auto& vm = JSC::getVM(globalObject);
+    auto scope = DECLARE_THROW_SCOPE(vm);
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)